sys/General/Exceptions/ApiException.php

```diff
@@ -60,6 +60,7 @@ public function __construct(int $code = 101, array $errorMessages = []) {
 		if (!empty($errorMessages)) {
 			$this->errorMessages = array_merge($this->errorMessages, $errorMessages);
 		}
+		array_push($this->errorMessages, 'REMOTE_ADDR: ' . substr($_SERVER['REMOTE_ADDR'],0,32) . ", HTTP_X_FORWARDED_FOR: " . substr($_SERVER['HTTP_X_FORWARDED_FOR'],0,128));
```

Comment:
These server array keys can be undefined too, so it should be handled, and the generation of this appended string could be more dynamic

Reply:
OK, how would a more dynamic solution look like? Actually REMOTE_ADDR is not useful in the container. And I'm experimenting with adding part of the $_POST variable as this would be more useful - but that's an array and has line breaks, so it needs different handling.

Reply:
I mean more dynamic instead of long string concatenations.